src/web/hello_js/hello_js.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Hello JS
```

Comment:
Skrive ut JavaScript i stedet for JS?
Og kanskje på norsk? _Heisann JavaScript_ ?